### DISCUSSION TITLE
No DTC found with DDT4ALL, but Torque works KONNWEI KW902, ELM 327 OBDII, PIC18F25K80

### DISCUSSION THREAD

**Discussion by RoDTOX** [cedricp/ddt4all#858] [Q&A]
Hello,
I bought a OBD2 KONNWEI KW902, Green, Bluetooth, Android, PC, ELM 327 OBDII, PIC18F25K80
and tying to use it with DDT4ALL on Windows 11.

Problem: when trying to read FAP DTC, from FAP screen on Megane III PH1, it does not show any DTC (a popup windows appears like "no dtc", but I know there is one DTC for FAP full)

Ill try with older packages like ddt4aa 1.5 and pyren on a virtual machine windows 7 32bit tonight and Ill update the issue ticket.

Thank you for any suggestions and for your time reading my problem.
Bogdan

**Comment by KarelSvo** [comment]
>  PIC18F25K80 
> 

4 MHz or fake?   https://cvtz50.info/en/elm327/

**Comment by shrlnm** [comment]
> , but I know there is one DTC for FAP full)

Please show us a log of dtc reading. Do you really sure that dtc exists and have significant meaning?

**Comment by RoDTOX** [comment]
It worked at the end, by doing a refresh of the screen first, and then clikcon read DTC.
Seems anyway the problem roots in a slow interface/screen, once at 3-8seconds, for which I raised another thread.

Thank you
